### PR TITLE
[new locale] tk: Add Turkmen

### PR DESCRIPTION
**Pull request by atamyratabdy**

### CODE REVIEW

File: src/locale/tk.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    70: "'inji",
+    80: "'inji",
+    2: "'nji",
+    7: "'nji",
+    20: "'nji",
+    50: "'nji",
+    3: "'ünjü",
+    4: "'ünji",
+    100: "'ünji",
+    6: "'ncy",
```
Comment (gnurgeldiyev) [inline src/locale/tk.js:20]: It should be "'njy"

File: src/locale/tk.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        L: 'DD.MM.YYYY',
+        LL: 'D MMMM YYYY',
+        LLL: 'D MMMM YYYY HH:mm',
+        LLLL: 'dddd, D MMMM YYYY HH:mm',
+    },
+    calendar: {
+        sameDay: '[bugün sagat] LT',
+        nextDay: '[ertir sagat] LT',
+        nextWeek: '[indiki] dddd [sagat] LT',
+        lastDay: '[düyn] LT',
```
Comment (gnurgeldiyev) [inline src/locale/tk.js:50]: It should be '[düýn] LT'

File: src/locale/tk.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        switch (period) {
+            case 'd':
+            case 'D':
+            case 'Do':
+            case 'DD':
+                return number;
+            default:
+                if (number === 0) {
+                    // special case for zero
+                    return number + "'inji";
```
Comment (gnurgeldiyev) [inline src/locale/tk.js:79]: It should be "'unjy"

File: src/locale/tk.js (hunk 1 above)
Comment (atamyratabdy) [reply src/locale/tk.js:20]: Yes, exactly. I will fix it. Thanks

File: src/locale/tk.js (hunk 2 above)
Comment (atamyratabdy) [reply src/locale/tk.js:50]: Yes, exactly. I will fix it. Thanks

File: src/locale/tk.js (hunk 3 above)
Comment (atamyratabdy) [reply src/locale/tk.js:79]: Yes, exactly. I will fix it. Thanks

**Comment by atamyratabdy** [comment]
Hi @icambron . I would like to add Turkmen language on moment.
Can you merge my pull request, please ?

**Comment by icambron** [comment]
I don't merge these. That would be @marwahaha or @ichernev

**Comment by marwahaha** [comment]
Hi @atamyratabdy , nice work!

Please add a file `src/test/locale/tk.js` which tests the changes you wrote. 
You can copy from `src/test/locale/en-gb.js` or similar.

**Comment by atamyratabdy** [comment]
Hi @marwahaha, thanks for confirmation.

I have added tests for my locale. However, there is a mismatch on versions, can you please update and merge my pull request, please. Thanks 🙋🙏.

**Comment by marwahaha** [comment]
Hi @atamyratabdy , unfortunately some tests that you added are failing. 
Could you investigate and adjust the files so the tests pass?

**Comment by atamyratabdy** [comment]
Hi @marwahaha, I have fixed failing tests and committed the fix.

Can you try to merge again and inform me if there needs any more adjustments.
Thanks 🙋🙏.

**Comment by marwahaha** [comment]
Great work @atamyratabdy

**Comment by atamyratabdy** [comment]
Thank you @marwahaha ✌.

**Pull request merged**